postcss.config.js

```diff
@@ -13,14 +13,19 @@ module.exports = {
     calc(),
     presetEnv({
       stage: 0,
+      features: {
+        'color-mod-function': {
+          unresolved: 'ignore',
```

Comment:
Is it good to ignore it?

Reply:
I do not know but I used this function only once for this moment.

Reply:
I would prefer `warn` level at least.